ngraph/test/onnx/onnx_import_quant.in.cpp

```diff
@@ -402,6 +402,274 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_quant_conv_linear_onnx_example) {
     test_case.run();
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_matmul_integer_2d_simple_zero_point) {
+    auto function = onnx_import::import_onnx_model(file_util::path_join(SERIALIZED_ZOO, "onnx/matmul_integer.onnx"));
+
+    auto test_case = test::TestCase<TestEngine>(function);
+
+    // don't change style for better readibility
+    // clang-format off
+    test_case.add_input(std::vector<uint8_t>{11, 7, 3,
+                                             10, 6, 2,
+                                              9, 5, 1,
+                                              8, 4, 0});                        // A
+    test_case.add_input(std::vector<uint8_t>{1, 4,
+                                             2, 5,
+                                             3, 6});                            // B
+    test_case.add_input(std::vector<uint8_t>{12});                              // a_zero_point
+    test_case.add_input(std::vector<uint8_t>{0});                               // b_zero_point
+
+    test_case.add_expected_output({4, 2}, std::vector<int32_t>{-38,  -83,
+                                                               -44,  -98,
+                                                               -50, -113,
+                                                               -56, -128});     // Y
+    // clang-format on
+    test_case.run();
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_matmul_integer_int8) {
+    auto function = onnx_import::import_onnx_model(file_util::path_join(SERIALIZED_ZOO, "onnx/matmul_integer_int8.onnx"));
+
+    auto test_case = test::TestCase<TestEngine>(function);
+
+    // don't change style for better readibility
+    // clang-format off
+    test_case.add_input(std::vector<int8_t>{-3,  7, 5, -6,
+                                             4, -5, 8,  7});                            // A
+    test_case.add_input(std::vector<int8_t>{ 5, -3,  7,  8,
+                                            -6, -8, -3,  6,
+                                             7,  9,  9, -5,
+                                             8,  7, -6,  7});                           // B
+    test_case.add_input(std::vector<int8_t>{5});                                        // a_zero_point
+    test_case.add_input(std::vector<int8_t>{5});                                        // b_zero_point
+
+    test_case.add_expected_output({2, 4}, std::vector<int32_t>{-55,  16, 89, -44,
+                                                               122, 154, 68, -39});     // Y
+    // clang-format on
+    test_case.run();
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_matmul_integer_complex_zero_point) {
```

Comment:
Complex might be misleading here (meaning a complex number which is not an allowed data type for this op). I suggest renaming it to something like "non_scalar" or "multilple".

Reply:
I changed it to "vectorized", since it is more accurate and should not lead to misunderstandings. What do you think?

Reply:
Yep, vectorized is ok. You will need to update the model file name too because the one that the test is looking for does not exist now.

Reply:
Ah, I did not notice it, thank you very much! Added missing model prototxt.